daru.gemspec

```diff
@@ -74,6 +74,8 @@ EOF
   spec.add_development_dependency 'ruby-prof'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'nokogiri'
-  spec.add_development_dependency 'guard-rspec'
   spec.add_development_dependency 'gruff'
+  if RUBY_VERSION >= '2.2.5'
```

Comment:
I'm not sure if this is valid for gemspecs. Have a look at the bundler docs for conditional dependencies. I vaguely remember running into problems once when I had done something similar.